justfile

```diff
@@ -70,7 +70,8 @@ typos:
   set -eo pipefail
 
   git_ls_files="$(git ls-files)"
-  git_ls_nonbinary_files="$(echo "$git_ls_files" | xargs file --mime | grep -v "; charset=binary" | cut -d: -f1)"
+  git_ls_nonbinary_files="$(echo "$git_ls_files" |  grep -v -E "^db/|\.png\$|\.ods\$")"
```

Comment:
This regex will be slightly annoying to maintain, does anything bad happen if there are binary files in that list?

Reply:
Some lints like `typos` will complain that they see strings they don't like, pointing us to update the list.

I wanted to avoid manual list, but `xargs file --mime | grep -v "; charset=binary"` costs around a second on every `git commit` on my machine.